_posts/2023/2023-01-22-only-use-locals-in-view-partials.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Only use locals in view partials"
+description: "It helps keep your state manageable"
+layout: article
+category: ruby
+image:
+  base: '2023/only-use-locals-in-view-partials'
+  alt: "Local building sign"
+  credit: "Priscilla Du Preez"
+  source: "https://unsplash.com/photos/acNPOikiDRw"
+
+---
+
+The Rails view layer is incredibly flexible, but can easily become tricky to manage even though it’s the main way our users and customers access our work.
```

Comment:
```suggestion
The Rails view layer, the main way our users and customers access our work, is incredibly flexible, but it can easily become tricky to manage.
```